build.gradle

```diff
@@ -49,8 +49,8 @@ configure(allProjs) {
     sourceCompatibility = 1.8
     targetCompatibility = 1.8
 
-    // Max: Set this to the main class for cli.
-    // Could not figure out how to configure it just for the cli subproject
+    // Set this to the main class for cli.
```

Comment:
:-)

Reply:
There is currently no other way. I also checked ;)